src/main/java/hw_8/Program.java

```diff
@@ -0,0 +1,28 @@
+package hw_8;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Program {
+    public static void main(String[] args) {
+
+        MyBrowser browser = new MyBrowser();
+        MyFile file = new MyFile();
+
+        Lecture30.finishWork(browser);
+        Lecture30.finishWork(file);
+
+        SessionCloser sessionCloser = (entities) -> {
```

Comment:
"(" and ")" can be removed if only one argument is used

Reply:
fixed